x-pack/plugins/security_solution/public/common/containers/sourcerer/use_signal_helpers.test.tsx

```diff
@@ -93,4 +117,73 @@ describe('useSignalHelpers', () => {
       expect(result.current.pollForSignalIndex).not.toEqual(undefined);
     });
   });
+
+  test('Init happened and signal index does not have data yet, poll function becomes available but createSourcererDataView throws an abort error', async () => {
```

Comment:
Can you update the description to be a bit clearer as to what expectation we're testing here? Are we testing that an error is called on abort?